core/arch/arm/kernel/boot.c

```diff
@@ -798,22 +799,43 @@ static int add_optee_dt_node(struct dt_descriptor *dt)
 		 * these.
 		 *
 		 * An SPI type of interrupt is indicated with a 0 in the
-		 * first cell.
+		 * first cell. A PPI type is indicated with value 1.
 		 *
 		 * The interrupt number goes in the second cell where
-		 * SPIs ranges from 0 to 987.
+		 * SPIs ranges from 0 to 987 and PPI ranges from 0 to 15.
 		 *
 		 * Flags are passed in the third cell where a 1 means edge
-		 * triggered.
+		 * triggered for SPI types and (8|mask<<8) means a lowlevel
```

Comment:
s/lowlevel/level/
If you want to go into more detail please do so, but then we should do the same for the edge triggered.

Reply:
ok, i'll more less verbose. The info are in the DT bindings.